src/storage/src/table/cell_based_table.rs

```diff
@@ -322,24 +322,15 @@ fn generate_column_id(column_descs: &[ColumnDesc]) -> Vec<ColumnId> {
 // (st1page): Maybe we will have a "ChunkIter" trait which returns a chunk each time, so the name
 // "RowTableIter" is reserved now
 pub struct CellBasedTableRowIter<S: StateStore> {
-    keyspace: Keyspace<S>,
-    /// A buffer to store prefetched kv pairs from state store
-    buf: Vec<(Bytes, Bytes)>,
-    /// The idx into `buf` for the next item
-    next_idx: usize,
-    /// A bool to indicate whether there are more data to fetch from state store
-    done: bool,
-    /// An epoch representing the read snapshot
-    epoch: u64,
+    /// An iterator that returns raw bytes from storage.
+    iter: StripPrefixIterator<S::Iter>,
```

Comment:
So both CellBasedTableRowIter(long-term) and CellBasedTableStreamingIter(short_term) do not need to scan, when can we just remove state_store scan?